finetuner/tuner/pytorch/miner.py

```diff
@@ -59,11 +62,151 @@ def mine(
         diffs = matches ^ 1
 
         matches.fill_diagonal_(0)
+
         triplets = matches.unsqueeze(2) * diffs.unsqueeze(1)
 
         return torch.where(triplets)
 
 
+class BaseClassEasyHardMiner:
```

Comment:
I think it would be flatter/more pythonic, if this was simply a function, called smth. along the lines of: `apply_easy_hard_strategy`?
Would you agree?

Reply:
Yes, I would do this